.storybook/Banner.js

```diff
@@ -735,7 +721,19 @@ storiesOf('Banner', module)
         text="red"
         bg="lightRed"
         iconName="star"
-        iconSize={26}
       />
     </Box>
   ))
+  .add('With text node', () => (
+    <Banner
+      textAlign="center"
+      mb={2}
+      p={2}
+      header="custom text"
+      text={
```

Comment:
Is there a good reason to create a custom prop rather than use `props.children` here? I think custom props like this only make sense when you want to poke "holes" into a component – e.g. `leftColumn`/`rightColumn`

Reply:
I'll allow either `text` or `children` since removing `text` at this point would be a breaking change.